contrib/forms/specific_question.py

```diff
@@ -0,0 +1,242 @@
+                        DEFAULT_MAX_NB_DOCUMENTS,
+                    ),
+                    mimetypes=configuration['configuration'].get(
+                        CleConfigurationItemFormulaire.TYPES_MIME_FICHIER.name
+                    ),
+                )
+
+            else:
+                raise ImproperlyConfigured(
+                    _('The type of the admission form item is unknown (%(type)s).' % {'type': form_item_type})
+                )
+
+            setattr(field, 'is_required', configuration['required'])
+            fields.append(field)
+            widgets.append(field.widget)
+
+        super().__init__(
+            fields=fields,
+            widget=ConfigurableFormItemWidget(
+                widgets=widgets,
+                field_configurations=self.field_configurations,
+                fields=fields,
+                is_bound=kwargs.pop('is_bound'),
+            ),
+            require_all_fields=False,
+            required=False,
+            **kwargs,
+        )
+
+    def clean(self, value):
+        """
+        Validate every value in the given list. A value is validated against
+        the corresponding Field in self.fields which will contain errors if any (errors attribute).
+        For example, if this MultiValueField was instantiated with
+        fields=(DateField(), TimeField()), clean() would call
+        DateField.clean(value[0]) and TimeField.clean(value[1]).
+        """
+        clean_data = []
+        has_error = False
+        for i, field in enumerate(self.fields):
+            try:
+                field_value = value[i]
+            except IndexError:
+                field_value = None
+            try:
+                clean_data.append(field.clean(field_value))
+            except ValidationError as e:
+                field.errors = e.error_list
+                has_error = True
+        if has_error:
+            raise ValidationError('')
+
+        out = self.compress(clean_data)
+        self.validate(out)
+        self.run_validators(out)
+        return out
+
+    def compress(self, data_list):
+        """Takes a list of valid values and returns a “compressed” version of those values – in a single value."""
+        compressed_data = self.initial
+        for index, data in enumerate(data_list):
+            field_configuration = self.field_configurations[index]
+            if field_configuration['type'] not in TYPES_ITEMS_LECTURE_SEULE:
+                compressed_data[str(field_configuration['uuid'])] = data
+
+        return compressed_data
+
+
+class ConfigurableFormMixin(forms.Form):
+    """Form whose some fields will be automatically created on the basis of a configuration."""
+
+    configurable_form_field_name = 'specific_question_answers'  # Name of the form field containing several values
+
+    def __init__(self, *args, **kwargs):
+        form_item_configurations = kwargs.pop('form_item_configurations', [])
```

Comment:
```suggestion
    def __init__(self, *args, form_item_configurations, **kwargs):
```
Ça permettrait d'avoir `form_item_configurations` en required keyword-only argument